resources/views/celebrities/create.blade.php

```diff
@@ -1,34 +1,70 @@
 <x-app-layout>
     <x-slot name="header">
-        Create Celebrity  
+        <h2 class="font-semibold text-xl leading-tight">
+            {{ __('Add a new celebrity to Database') }}
```

Comment:
My suggestion is to keep the public pages without any terms that might be confusing to other people. The database is a term in the IT industry, normal users might not know what that means. 

I would suggest changing this to Create a celebrity or Add a new celebrity...